source/slang/slang-lower-to-ir.cpp

```diff
@@ -5595,6 +5604,11 @@ struct DeclLoweringVisitor : DeclVisitor<DeclLoweringVisitor, LoweredValInfo>
         irWitnessTable->setOperand(0, irSubType);
 
         addLinkageDecoration(context, irWitnessTable, inheritanceDecl, mangledName.getUnownedSlice());
+
+        // TODO(JS):
+        // Not clear what to do here around HLSLExportModifier. 
+        // In HLSL it only (currently) applies to functions, so perhaps do nothing is reasonable.
+        
```

Comment:
Similarstatement here: I'm not sure why we'd need to process `export` modifiers on inheritance clauses...